Authenticator/Source/OTPAppDelegate.swift

```diff
@@ -47,8 +49,8 @@ class OTPAppDelegate: UIResponder, UIApplicationDelegate {
         // Restore white-on-black style
         SVProgressHUD.setDefaultStyle(.Dark)
 
-        tokenManager.delegate = self
-        let rootViewController = OTPTokenListViewController(tokenManager: tokenManager)
+        let rootViewController = TokenListViewController(viewModel: tokenList.viewModel, delegate: tokenList)
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 109 characters (line_length)